UVAtlas/isochart/UVAtlas.cpp

```diff
@@ -113,7 +113,9 @@ namespace
             }
         }
 
+#ifdef _MSC_VER
```

Comment:
Unfortunately, this doesn't work because suppress only works on the next code line.

This has to be made into a push, warning disable, pop construction.

Reply:
I hope I resolved the issue, but I don't use MSVC and I don't know where the #pragma warning(pop) should be.